src/main/java/org/cactoos/scalar/Or.java

```diff
@@ -34,6 +34,36 @@
 /**
  * Logical disjunction.
  *
+ * <p>This class can be effectively used to iterate through
+ * a collection, just like
+ * {@link java.util.stream.Stream#forEach(java.util.function.Consumer)}
+ * works:</p>
+ *
+ * <pre>
+ * new Or(
+ *    new ProcOf<>(input -> System.out.printf("\'%s\' ", input) ),
```

Comment:
@Vatavuk You can't use `<` or `>` inside javadoc without some kind of escaping, either `&lt;` and `&gt;` or preferably replace the `<pre>...</pre>` with `{@code ...}`